doc/doc-primitives.saty

```diff
@@ -118,11 +263,17 @@ document (|
       +command (`arabic`) (tI --> tS) {
         受け取った整数の十進文字列を返す．
       }
+      +command (`show-float`) (tF --> tS) {
+        受け取った浮動小数点数の十進文字列を返す．
+      }
       +command (`string-unexplode`) ((tLIST tI) --> tS) {
         受け取った整数列をUnicodeコードポイント列と看なして対応する文字列を返す．
         Unicodeコードポイントとして不適切な整数が含まれていた場合の動作は未定義．
         \subject-to-change;
       }
+      +command (`string-explode`) (tS --> (tLIST tI)) {
+        受け取った文字列をUnicodeコードポイント列として返す．
```

Comment:
> Unicodeコードポイント列

I'm just curious.  Do you mean a sequence of [Unicode code points](https://unicode.org/glossary/#code_point) or [Unicode scalar values](https://unicode.org/glossary/#unicode_scalar_value)?  I think it should be that of Unicode scalar values.